src/components/StickyPlayer/StickyPlayer.tsx

```diff
@@ -36,20 +45,30 @@ const StickyPlayer: React.FC<StickyPlayerProps> = ({ musicNft }) => {
         padding: ".7em 1em",
       }}
     >
-      <div style={{ display: "flex", flexDirection: "column" }}>
-        <Title style={{ margin: "0" }} level={5}>
-          {musicMetaData?.body.title}
-        </Title>
-        <Text type="secondary">{musicMetaData?.body.artist}</Text>
-      </div>
+      
       <audio
         autoPlay
         loop
         controls
         src={`${ipfsToHttps(musicNft.assetUri)}`}
       ></audio>
-    </div>
+    </Card>
   );
 };
 
 export default StickyPlayer;
+
+interface TitleNodeProps{
+  musicMetaData: MusicNftMetaData | undefined
+}
+
+const TitleNode: React.FC<TitleNodeProps> = ({musicMetaData})=>{
```

Comment:
Would be great if we outsource this in separate file so in future if code gets bigger it will easier to manage,
Lets keep it 1 component per file

Reply:
Yes I also believe so, but I think separating now would be an overkill since the components are tightly coupled. But it will definitely be moved to it's own file soon.